web-app/src/app/screens/Feed/index.tsx

```diff
@@ -57,6 +58,60 @@ export function formatProvidersSorted(provider: string): string[] {
   return providersSorted;
 }
 
+export function getFeedFormattedName(
+  sortedProviders: string[],
+  dataType: 'gtfs' | 'gtfs_rt',
+  feedName?: string,
+): string {
+  let formattedName = '';
+  if (sortedProviders[0] !== undefined && sortedProviders[0] !== '') {
+    formattedName += sortedProviders[0];
+  }
+  if (feedName !== undefined && feedName !== '') {
+    if (formattedName !== '') {
+      formattedName += ', ';
+    }
+    formattedName += `${feedName}`;
+  }
+  return formattedName;
+}
+
+export function generateDescriptionMetaTag(
```

Comment:
Let's think about how supporting multiple languages affects this implementation.